symplyphysics/laws/condensed_matter/current_density_from_mobility.py

```diff
@@ -0,0 +1,55 @@
+from sympy import (Eq, solve)
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+)
+
+# Description
+## Current density is the amount of charge per unit time that flows through a unit area of a chosen
+## cross section. The current density vector is defined as a vector whose magnitude is the electric
+## current per cross-sectional area at a given point in space, its direction being that of the motion
+## of the positive charges at this point. In SI base units, the electric current density is measured
+## in amperes per square metre.
+
+## Law is: j = q * (-n1 * u1 + n2 * u2) * E, where
```

Comment:
I see it is very close to https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/laws/condensed_matter/current_density_from_concentration_and_velocity_of_charge_carriers.py
Could you derive one law from another?

Reply:
Yes, I can get this law from the law j = qnv.  But in this case, the law is more specific, since it takes into account electrons and holes, which is necessary when considering semiconductors.

Reply:
> I see it is very close to https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/laws/condensed_matter/current_density_from_concentration_and_velocity_of_charge_carriers.py Could you derive one law from another?

Yes, I can get this law from the law j = qnv.  But in this case, the law is more specific, since it takes into account electrons and holes, which is necessary when considering semiconductors.

Reply:
You can do it in reverse order and derive j = qnv from this law.

Reply:
> You can do it in reverse order and derive j = qnv from this law.

Yes, I can. I do not understand how this applies to the code and what is the task associated with this remark.